tslint.json

```diff
@@ -1,140 +1,24 @@
-            "signatures": [
-                "localize",
-                "nls.localize"
-            ],
-            "keyIndex": 0,
-            "messageIndex": 1
-        }]},
-        "linebreak-style": false,
-        "quotemark": false,
-        "newline-per-chained-call": false,
-        "member-ordering": false,
-        "no-parameter-properties": false,
-        "no-inferrable-types": false,
-        "strict-type-predicates": true,
-        "increment-decrement": [true, "allow-post"],
-        "no-implicit-dependencies": false,
-        "ordered-imports": false,
-        "variable-name": false,
-        "prefer-readonly": false,
-        "comment-format": false,
-        "align": false,
-        "no-import-side-effect": false,
-        "switch-default": false,
-        "trailing-comma": false,
-        "switch-final-break": false,
-        "strict-string-expressions": false,
-        "newline-before-return": false,
-        "object-literal-sort-keys": false,
-        "no-suspicious-comment": false,
-        "export-name": false,
-        "no-magic-numbers": false,
-        "no-unsafe-any": false,
-        "no-unbound-method": false,
-        "detect-child-process": false,
-        "match-default-export-name": false,
-        "no-unnecessary-class": false,
-        "function-name": false,
-        "interface-name": false,
-        "cyclomatic-complexity": false,
-        "strict-comparisons": false,
-        "no-http-string": false,
-        "static-this": false,
-        "non-literal-fs-path": false,
-        "prefer-function-over-method": false,
-        "max-func-body-length": false,
-        "no-angle-bracket-type-assertion": false,
-        "no-for-in": false,
-        "arrow-return-shorthand": false,
-        "no-require-imports": false,
-        "no-string-literal": false,
-        "prefer-type-cast": false,
-        "no-non-null-assertion": false,
-        "ban-ts-ignore": false,
-        "no-namespace": false,
-        "no-unnecessary-local-variable": false,
-        "prefer-conditional-expression": false,
-        "no-reference": false,
-        "no-inner-html": false,
-        "no-single-line-block-comment": false
-    },
-    "rulesDirectory": ["node_modules/tslint-microsoft-contrib", "node_modules/tslint-no-unused-expression-chai/rules"],
-    "linterOptions": {
-        "exclude": [
-            "node_modules/**",
-            "out/**",
-            "src/test/**",
-            "src/explorer/**"
-        ]
-    }
+{
+	"rules": {
+		"class-name": true,
+		"no-console": [true, "log"],
+		"no-debugger": true,
+		"no-string-throw": true,
+		"no-var-keyword": true,
+		"prefer-const": true,
+
+		"file-header": [
+			true, 
+			{
+				"match": "Copyright \\(c\\) Microsoft Corporation\\. All Rights Reserved\\.\\nLicensed under the MIT License\\.",
+				"default" : "Copyright (c) Microsoft Corporation. All Rights Reserved.\nLicensed under the MIT License."
+			}
+		],
+		"indent": [true, "tabs"],
```

Comment:
>"tabs" [](start = 19, length = 6)

boo :P

Reply:
Sorry. :-) I would make the case this is more [accessible](https://www.reddit.com/r/javascript/comments/c8drjo/nobody_talks_about_the_real_reason_to_use_tabs/) but I'm game to adapt to your norms.

Reply:
Sorry. :-) I would make the case this is more accessible but I'm game to adapt to your norms.

---
In reply to: [430737144](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r430737144) [](ancestors = 430737144)

Reply:
Going track this issue with Gabe's comment about general style issues above. Thus closing this one.

---
In reply to: [436289686](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r436289686) [](ancestors = 436289686,430737144)